pkg/nsx/services/vpc/vpc.go

```diff
@@ -353,6 +353,7 @@ func (s *VPCService) getSharedVPCNamespaceFromNS(ns string) (string, error) {
 	}
 
 	// Retrieve the shared vpc namespace from annotation
+	// kube-system/infra-vpc
```

Comment:
Use comment as you added below
The format should be namespace/vpc_name

Reply:
ok